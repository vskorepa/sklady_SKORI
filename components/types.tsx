export type Row = {
  id: number;
  code: string;
  description?: string;
  count: number;
  name: string;
  storage: string;
};
