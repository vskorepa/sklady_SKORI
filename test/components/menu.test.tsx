import React from 'react'
import renderer from 'react-test-renderer'
import { Menu } from '../../components/menu'

describe('Index', () => {
  it('renders the html we want', async () => {
    console.log(renderer)
    const component = renderer.create(<Menu />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
