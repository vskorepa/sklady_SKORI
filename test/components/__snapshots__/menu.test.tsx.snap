// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders the html we want 1`] = `
<div
  className="menu__Wrapper-xmzi79-0 ZRMtO"
>
  Menu
</div>
`;
